src/components/organims/welcomeByOrg.js

```diff
@@ -0,0 +1,141 @@
+import React from 'react';
+import {Redirect, navigate} from '@reach/router';
+import AppMinimal from '../layout/AppMinimal';
+import Seo from '../molecules/Seo';
+import Button from '../atoms/Button';
+import styles from './welcome.module.css';
+import gql from 'graphql-tag';
+
+const getOrgInfo = gql`
+  query($slug: String!) {
+    organizationWithSlug(slug: $slug) {
+      slug,
+      name,
+      pictureUrl,
+    }
+  }
+`;
+
+export class WelcomeByOrg extends React.Component{
+  constructor(props){
+    super(props);
+    this.state = {
+      title: 'Your organization name',
+      description: 'Your organization description',
+      details: 'Your organization details',
+      email: 'name.org',
+      pictureUrl: '',
+      hasDownloaded: false,
+    }
+  }
+  componentWillMount(){
+    const { data, organizationSlug } = this.props;
+    const parsedUrl = new URL(window.location.href);
+    // If url contain ?dl=true just dl the file too !
+    if(parsedUrl && parsedUrl.searchParams.get('dl')){
```

Comment:
The 3rd test-case (direct DL) does not work on my side: it does DL but do not navigate to 2nd page.

Reply:
Right !